src/btree/bt_vrfy.c

```diff
@@ -537,7 +537,15 @@ __verify_tree(
      * (just created), it won't have a disk image; if there is no disk image, there is no page
      * content to check.
      */
-    if (page->dsk != NULL)
+    if (page->dsk != NULL) {
+        /* Validate the page disk write generation number against a parent's write generation
+         * number. */
```

Comment:
The comment format isn't quite right. It should be multi-line. It could also be expanded to describe why the parent should always be newer in the verify check.

Reply:
Fixed and added more details of why the parent write generation number must be higher.